rest_framework_swagger/introspectors.py

```diff
@@ -488,6 +499,19 @@ def build_query_parameters_from_django_filters(self):
         return params
 
 
+def get_primitive_type(var):
+    if isinstance(var, bool):
+        return 'boolean', 'boolean'
+    elif isinstance(var, int):
+        return 'integer', 'int64'
+    elif isinstance(var, float):
+        return 'number', 'float'
+    elif isinstance(var, six.string_types):
```

Comment:
This code is from upstream, I would rather leave it as close to vanilla as possible to reduce future merge pains